ctk/common/src/main/kotlin/org/codice/compliance/verification/core/RequestVerifier.kt

```diff
@@ -25,8 +25,8 @@ abstract class RequestVerifier(private val samlRequestDom: Node) : CoreVerifier(
 
     /** 3.2.1 Complex Type RequestAbstractType */
     override fun verify() {
-        super.verify()
         verifyRequestAbstractType()
+        super.verify()
```

Comment:
In one of the unit tests we pass in an empty string as the version and that was failing on a non-saml compliance related exception because the `SamlVersioningVerification` was running before the version field was validated by `CommonDateTypeVerifier`.